<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWStatisticsDocker</class>
 <widget class="QWidget" name="KWStatisticsDocker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>94</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="statistics">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Words</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sentences</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Syllables</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lines</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Characters incl. spaces</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Characters without spaces</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>East Asian characters</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flesch reading ease</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="count">
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="refresh">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="autoRefresh">
     <property name="text">
      <string>Automatically update</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="footEndNotes">
     <property name="text">
      <string>Only count main text</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
